oxidation/crates/parsec_core_fs/src/storage/local_database.rs

```diff
@@ -0,0 +1,44 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+use diesel::connection::SimpleConnection;
+use diesel::r2d2::{ConnectionManager, Pool, PooledConnection};
+use diesel::SqliteConnection;
+use std::path::PathBuf;
+
+use crate::error::{FSError, FSResult};
+
+pub const SQLITE_MAX_VARIABLE_NUMBER: usize = 999;
```

Comment:
Should add a comment to explain what this is for, and the rational for this uncommon value